index.html

```diff
@@ -1,6 +1,7 @@
 ---
 layout: landing
 title: Asylo
+keywords: google,asylo,opensource framework,flexible framework,opensource application,asylo-site 
```

Comment:
Why have explicit keywords here when it should inherit like the other pages?

Reply:
for front matter defaults we need to add it on the index.html page